### PR TITLE
[cas-432] New MessageListView attrs

### PR DESCRIPTION
**Pull request by samiuelson**
https://stream-io.atlassian.net/browse/CAS-432

### Description

Add `streamCopyMessageActionEnabled`, `streamFlagMessageActionEnabled`, and `streamStartThreadMessageActionEnabled` attrs to `MessageListView`

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Changelog updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Reviewers added

### CODE REVIEW

**Comment by zsmb13** [comment]
If I disable all of these, and long press on someone else's message (where Edit and Delete actions aren't available), I end up with this slightly odd UI:

Screenshot_20201113_101447

Not sure if it's worth addressing, just wanted to bring it up.

File: stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt [hunk 1]
```diff
@@ -87,15 +88,23 @@ internal class MessageMoreActionDialog(
...
-        binding.startThreadButton.setOnClickListener {
-            onStartThreadHandler(message)
-            dismiss()
+
+        binding.startThreadButton.apply {
+            isVisible = style.startThreadMessageActionEnabled
+            setOnClickListener {
+                onStartThreadHandler(message)
+                dismiss()
+            }
```
Comment (zsmb13) [inline stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt:97]: Nit: we could skip setting up the listener if the View is hidden

File: stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt [hunk 2]
```diff
@@ -73,6 +73,7 @@ internal class MessageMoreActionDialog(
                 dismiss()
             }
         }
+        binding.flagMessageButton.isVisible = style.flagMessageActionEnabled
```
Comment (zsmb13) [inline stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt:76]: This change now shows the Flag action on our own messages as well (if the action is enabled), overriding the `visibility = GONE` call a couple lines above

**Comment by samiuelson** [comment]
> If I disable all of these, and long press on someone else's message (where Edit and Delete actions aren't available), I end up with this slightly odd UI:
> 
> 
> 
> Not sure if it's worth addressing, just wanted to bring it up.

I removed those extra spacing elements. I don't think they are up to date to UI requirements and the UI looks good without them.

File: stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt [hunk 3]
```diff
@@ -55,13 +55,14 @@ internal class MessageMoreActionDialog(
         setCanceledOnTouchOutside(true)
         binding.startThreadButton.isVisible = canThreadOnMessage()
         binding.copyMessageButton.isVisible = canCopyOnMessage()
-        if (isMessageCreatedByCurrentUser()) {
+        if (isMessageNotCreatedByCurrentUser()) {
```
Comment (zsmb13) [inline stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt:58]: Really interesting to see that this was the other way around 😄

**Review by zsmb13 — APPROVED** [review]
Looks all good to me! 👍

**Review by JcMinarro — APPROVED** [review]

File: stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt (hunk 3 above)
Comment (samiuelson) [reply stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt:58]: 🙉  This was quite confusing!

**Pull request merged**